src/scripts/background.js

```diff
@@ -1986,6 +2011,9 @@ window.TogglButton = {
       console.log('Load content script: [' + file + ']');
     }
     browser.tabs.insertCSS(tabId, { file: 'styles/style.css' })
+      .then(function () {
+        return browser.tabs.insertCSS(tabId, { file: 'styles/edit-form.css' });
+      })
       .then(function () {
         browser.tabs.insertCSS(tabId, { file: 'styles/autocomplete.css' });
```

Comment:
For consistency and correcting the promise chain I'd recommend retuning here as well
```suggestion
        return browser.tabs.insertCSS(tabId, { file: 'styles/autocomplete.css' });
```